src/main/java/dev/ebullient/convert/tools/dnd5e/Json2QuteCommon.java

```diff
@@ -1099,4 +1099,27 @@ static PrereqFields fromString(String name) {
         }
         return PrereqFields.unknown;
     }
+
+    enum AbilityScoreIncreaseFields implements JsonNodeReader {
+        str,
+        dex,
+        con,
+        intel,
+        wis,
+        cha,
+        choose,
+        unknown // catcher for unknown attributes (see #fromString())
+    }
+
+    static AbilityScoreIncreaseFields abilityScoreIncreaseFieldFromString(String name) {
+        for (AbilityScoreIncreaseFields field : AbilityScoreIncreaseFields.values()) {
+            if (field.name().equals(name))
```

Comment:
equalsIgnoreCase ?

Reply:
It was to allow me to handle int with a special case.  I can probably do it another way as well.  This was just how I thought it up at the time.